### ISSUE TITLE
Cannot use with league/flysystem-sftp:^3.0

### ISSUE THREAD

**Issue by nhung-le** [1up-lab/OneupFlysystemBundle#274]
### Bug Report

|    Q        |   A
|------------ | ------
| BC Break    | yes
| Version     | 4.5.0

#### Summary

```
Package league/flysystem-sftp is abandoned, you should avoid using it. Use league/flysystem-sftp-v3 instead.
```
So I installed league/flysystem-sftp-v3 using composer. But I can't do last step cache clear anymore because it keeps saying:
```
In getOneupFlysystem_MediaFsFilesystemService.php line 25:

  Class "League\Flysystem\PhpseclibV2\SftpAdapter" not found
```

#### How to reproduce

- Uninstall your old `league/flysystem-sftp ` and install new `league/flysystem-sftp-v3 ` you will see the above error.
- I took a look in codes and see in https://github.com/1up-lab/OneupFlysystemBundle/blob/main/src/Resources/config/adapters.xml#L29
```
        <service id="oneup_flysystem.adapter.sftp" class="League\Flysystem\PhpseclibV2\SftpAdapter" abstract="true" public="false">

```
- that OneupFlysystemBundle still load `League\Flysystem\PhpseclibV2\SftpAdapter` which will cause this issue. because in the new bundle `league/flysystem-sftp-v3` it should be `League\Flysystem\PhpseclibV3\SftpAdapter`. 
- Other file paths should be the same since the biggest problem is `flysystem-sftp-v3` requires [phpseclib/phpseclib](https://packagist.org/packages/phpseclib/phpseclib): ^3.0

If you could support this, thank you very much!! 🙏

**Comment by bytehead** [comment]
It will be fixed with #272

**Comment by bytehead** [comment]
Duplicate of #270.
